src/bytecode/astgen.rs

```diff
@@ -284,9 +289,57 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
         dest
     }
 
+    fn visit_expr_assert(&mut self, expr: &ExprCallType, _dest: DataDest) {
```

Comment:
The complete functions seems to be a little complicated or at least has some potential for optimization.